src/main/java/net/smartcosmos/cluster/auth/AuthServerApplication.java

```diff
@@ -59,6 +62,7 @@
 @SessionAttributes("authorizationRequest")
 @Slf4j
 @EnableDiscoveryClient
+@EnableSmartCosmosMonitoring
 public class AuthServerApplication extends WebMvcConfigurerAdapter {
```

Comment:
This seems to be a more complex task in auth-server, because there are some methods that override methods from `WebMvcConfigurerAdapter`.